func/ui.h

```diff
@@ -3,4 +3,4 @@ int requestRecipeName(Recipe* recipes, int recipeCount);
 int requestAmountOfPeople();
 void printListOfAlternativeRecipes(Recipe[]);
 int requestRecipeNumber();
-void printRecipe(const Recipe *, const int);
+void printRecipe(Recipe*, const int);
```

Comment:
```suggestion
void printRecipe(const Recipe*, const int);
```
Hvorfor fjerner I en forbedring fra #38?